internal/services/recoveryservices/recovery_services_vault_resource_test.go

```diff
@@ -492,6 +492,11 @@ resource "azurerm_recovery_services_vault" "test" {
   }
 
   soft_delete_enabled = false
+
+  timeouts {
+    create = "60m"
+    update = "60m"
+  }
```

Comment:
should we instead be updating the default timeouts

Reply:
I'm not sure, it only timed out in the `TestAccRecoveryServicesVault_identity` test case.

Reply:
yes but this doesn't fix it for any users who encounter it? could we please update the resource not test?

Reply:
copy that, thank you.